lib/DB/Schema/Result/PoolProblem.pm

```diff
@@ -3,10 +3,50 @@ use base qw/DBIx::Class::Core/;
 use strict;
 use warnings;
 
-### this is the table that stores pool problems.
+=head1 DESCRIPTION
+
+This is the database schema for a Pool Problem, which is a problem in a Problem Pool.
+
+=head2 fields
+
+=over
+
+=item *
+
+C<pool_problem_id>: database id (primary key, autoincrement integer)
+
+=item *
+
+C<problem_pool_id>: the database id of the problem pool the problem  is in (foreign key)
+
+=item *
+
+C<params>: a object that holds the information about the pool problem
+
+=over
+
+=item *
+
+C<library_id>: a database id of the problem (foreign key)
+
+=item *
+
+C<problem_path>: alternatively, the path of the problem
+
+=back
+
+=head4
```

Comment:
I think you are going to need to drop the `=head4` here.  Headers can't be inside an item list.  Either that or move the `=back` below above this line, but then the note is not at the same level.

In any case, this causes a POD error.

Reply:
fixed.